cdk-infra/package.json

```diff
@@ -9,12 +9,13 @@
     "watch": "tsc -w",
     "test": "jest",
     "cdk": "cdk",
-    "deploy:enhanced": "cdk deploy -c enhanced=true -c customFeatureName=enhancedApp -c env=dotcomstg --require-approval never --all",
-    "deploy:enhanced:worker": "cdk deploy auto-builder-stack-enhancedApp-worker -c enhanced=true -c customFeatureName=enhancedApp -c env=dotcomstg --require-approval never --exclusively",
-    "deploy:enhanced:webhooks": "cdk deploy auto-builder-stack-enhancedApp-webhooks -c enhanced=true -c customFeatureName=enhancedApp -c env=dotcomstg --require-approval never --exclusively",
-    "deploy:feature": "cdk deploy -c enhanced=true -c env=dotcomstg --require-approval never --all",
-    "deploy:feature:worker": "cdk deploy  -c enhanced=true -c env=dotcomstg --require-approval never --exclusively",
-    "deploy:feature:webhooks": "cdk deploy -c enhanced=true -c env=dotcomstg --require-approval never --exclusively"
+    "deploy:enhanced:prd": "cdk deploy -c enhanced=true -c customFeatureName=enhancedApp -c env=prd --require-approval never --all",
```

Comment:
Sorry, I should have considered this in the other thread, but would changing the names of the commands here impact the GH actions command calls? Do we need to have separate ones per env? Or is there something we could do like `npm run deploy:enhanced -- -c env=[prd|stg|dotcomstg|etc]`?

Reply:
Hi Raymund! We could absolutely do something like you mentioned here, and that would probably be more straightforward as there will be fewer commands in the `package.json`